src/drakvuf.cpp

```diff
@@ -205,10 +205,15 @@ static bool is_interrupted(drakvuf_t drakvuf, void*)
     return drakvuf_is_interrupted(drakvuf);
 }
 
+int drakvuf_c::is_interrupted()
+{
+    return drakvuf_is_interrupted(drakvuf);
+}
+
 void drakvuf_c::loop(int duration)
 {
     GThread* timeout_thread = startup_timer(this, duration);
-    drakvuf_loop(drakvuf, is_interrupted, nullptr);
+    drakvuf_loop(drakvuf, ::is_interrupted, nullptr);
```

Comment:
These changes seem unrelated and I'm not sure why we would need two different versions of `is_interrupted`

Reply:
One version is member function of drakvuf_c.
Another version is implementation-specific global function needed for `drakvuf_loop`.

Without "::" this code would not compile because of name overloading rules of c++. This means "I need a global function with this name, not member one".